### PR TITLE
Added Margin Info endpoint & claim position

### PR DESCRIPTION
**Pull request by itsdeka**
### Description:
Implemented Margin Info endpoint and claim position

### Breaking changes:
- []

### New features:
- [X]

### Fixes:
- [ ]

### PR status:
- [X] Version bumped
- [X] Change-log updated

### CODE REVIEW

File: bfxapi/models/margin_info.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        """
+        symbol = raw_margin_info[1]
+        tradable_balance = raw_margin_info[2][MarginInfoModel.TRADABLE_BALANCE]
+        gross_balance = raw_margin_info[2][MarginInfoModel.GROSS_BALANCE]
+        buy = raw_margin_info[2][MarginInfoModel.BUY]
+        sell = raw_margin_info[2][MarginInfoModel.SELL]
+        return MarginInfo(symbol, tradable_balance, gross_balance, buy, sell)
+
+    def __str__(self):
+        return "Margin Info {}".format(self.symbol)
```
Comment (robertkowalski) [inline bfxapi/models/margin_info.py:46]: maybe we can print more infos here?

```py
return "Margin Info {} buy={} sell={} tradable_balance={} gross_balance={}".format(self.symbol, self.buy, self.sell, self. tradable_balance, self. gross_balance)
```

**Review by robertkowalski — CHANGES_REQUESTED** [review]
lgtm, one small improvement for the string fromatter

File: bfxapi/models/margin_info.py (hunk 1 above)
Comment (itsdeka) [reply bfxapi/models/margin_info.py:46]: thanks

**Review by robertkowalski — APPROVED** [review]
lgtm

**Pull request merged**